lmdb-sys/src/bindings.rs

```diff
@@ -1,57 +1,57 @@
 /* automatically generated by rust-bindgen */
```

Comment:
Should this file be formatted/modified despite its claim to be auto-generated here?

Reply:
Well, they actively avoided formatting it, and it looks fine. That's why I added an exception for it.